### ISSUE TITLE
Fail to deconstruct VCF from the PGGBGFA file

### ISSUE THREAD

**Issue by isaamael** [pangenome/pggb#429]

**Comment by isaamael** [comment]
> @isaamael, it looks like you're right, `vg deconstruct` doesn't finish on time and doesn't write anything! Can you try to request more time for the job?

Hello sir,

Thanks for your suggestions.

In theory, there should be no time limit on the SLURM system I'm using, so I'm not sure why I'm encountering timeout issues. After manually defining the runtime, I generated some low-confidence VCF, which I attribute to the interference caused by the low-quality genome during the GFA construction.

In any case, after filtering the original input genome, I think it would be best to rerun pggb and generate the VCF directly from the pipeline. However, I tried the following command and encountered an error:
```
pggb -i $pggbdir/sly12.fa \
     -o $pggbdir/sly12.fa.out \
     -p 5000 -l 25000 -p 90 -n 2 -K 19 -F 0.001 -t 20 \
     -k 19 -f 0 -B 10000000 \
     -j 0 -e 0 -G 700,900,1100 -P 1,19,39,3,81,1 -O 0.001 -d 100 -Q Consensus_ \
     --vcf-spec 'solyc#0#12'

```
```
[vg::deconstruct] making VCF with reference=solyc#0#12 and delim=# xxxxxxxxxxxxx solyc#0#12 ------------ 0
Error [vg deconstruct]: No specified reference path or prefix found in graph
Command exited with non-zero status 1
```
My reference is indeed `solyc#0#12`, and the path in the output GFA is the same. I'm unclear about the correct way to specify `--vcf-spec` parameter and how to define `LEN`.

Some beginner questions and would appreciate your help : )

**Comment by AndreaGuarracino** [comment]
Just specify "... --vcf-spec solyc" and it should work!

________________________________

**Comment by isaamael** [comment]
i will try it !

**Comment by isaamael** [comment]
Hi sir, @AndreaGuarracino 
i try cmd
```
pggb -i $pggbdir/sly12.fa \
     -o $pggbdir/sly12.fa.out \
     -p 5000 -l 25000 -p 90 -n 10 -K 19 -F 0.001 -t 20 \
     -k 19 -f 0 -B 10000000 \
     -j 0 -e 0 -G 700,900,1100 -P 1,19,39,3,81,1 -O 0.001 -d 100 -Q Consensus_ \
     --vcf-spec solyc

```
and report an error

```
[wfmash::align::computeAlignments] aligned 78.50% @ 2.94e+05 bp/s elapsed: 00:02:17:56 remain: 00:00:37:46[E::fai_retrieve] Failed to retrieve block: unexpected end of file
Command terminated by signal 11
```

Did i make some mistakes ?

**Comment by AndreaGuarracino** [comment]
Oh no! It seems your FASTA file is corrupted. Can you check that your input FASTA is healthy?

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by isaamael** [comment]
yes! indeed 
I denove generated the FASTA and performed pggb.
Now, GFA, VCF, and I are all as good as expected :)
Sincere thanks, thanks for your help!

Additionally, do the numbers in the GT field of the VCF point to the paths of AT?

**Comment by AndreaGuarracino** [comment]
Yes. The GT numbers are allele indices (0 = REF, 1 = first ALT, 2 = second ALT, …), and the AT field lists one traversal per allele in that same order.
